common/src/main/java/org/conscrypt/OpenSSLX509CRL.java

```diff
@@ -415,7 +415,9 @@ public String toString() {
     protected void finalize() throws Throwable {
         try {
             if (mContext != 0) {
-                NativeCrypto.X509_CRL_free(mContext, this);
+                long toFree = mContext;
```

Comment:
nit: one read should be enough, isn't it?

Reply:
It's more so we can zero out the field before we call the JNI code.  Which also _ought_ not to be necessary but I'm getting paranoid... WDYT?

Reply:
What I mean is that `mContext` is read twice: in the if statement and in `toFree = mContext`. Both reads should return the same value as It's nullified only in `finalize` and only one thread calls it, so it probably does not really matter. Hence just a nit.

Reply:
Oh, good point... I'll change that.

Reply:
Done!